modules/ROOT/pages/_partials/supported-versions.adoc

```diff
@@ -113,21 +114,24 @@ Comparing this to the xref:1.4@csharp.adoc#supported-versions[supported versions
 .Support Notice - Apple macOS
 CAUTION: macOS is supported ONLY for testing and development purposes.
 
-:msg_title: Apple macOS
-:msg_component: OSX 10.11
-:msg_release: 2.8
-include::{root-partials}deprecationNotice.adoc[]
+.Deprecated versions
```

Comment:
Camel case "Versions"?

Reply:
I thought table captions were sentence case?

Reply:
Change done locally  and force-pushed